pkg/generate/crossplane/crossplane.go

```diff
@@ -169,5 +168,21 @@ func Crossplane(
 			}
 		}
 	}
+
+	// Finally, generate the single hooks.go file from one of the CRDs (doesn't
```

Comment:
We actually generate a `hooks.go` for every controller, hence for every CRD in that service. The only difference between other controller files is that `hooks.go` is not generated if it already exists because it's meant to be only a boilerplate to be edited by the developer after the code generation step and we don't want to override their custom code.